flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1473,6 +1499,8 @@ def test_attach_volume_validate_size(self):
             new_volume.blockdevice_id, attach_to=self.this_node,
         )
 
+        self.assertNotEqual(attached_volume, None)
```

Comment:
i was trying to save cpu cycles attempting a future operation using ``attached_volume`` when it is guaranteed to not succeed.